cmake/SSGCommon.cmake

```diff
@@ -641,6 +641,20 @@ macro(ssg_make_html_stats_for_product PRODUCT)
     )
 endmacro()
 
+macro(ssg_render_policies_for_product PRODUCT CONTROL_FILES)
+        foreach(CONTROL_FILE IN LISTS CONTROL_FILES)
+        add_custom_command(OUTPUT "${CMAKE_BINARY_DIR}/${PRODUCT}/rendered-policies/${CONTROL_FILE}.html"
```

Comment:
Nitpick: move the OUTPUT parameter to the line below.

Reply:
fixed